src/components/Tabs/Annotation/AnnotationTab.tsx

```diff
@@ -105,64 +108,70 @@ export default function AnnotationTab(props: AnnotationTabProps): ReactElement {
-              console.log(csvData);
-            }}
-          >
-            Export as CSV
-          </Button>
+        <TextButton
+          onClick={() => {
+            const csvData = props.annotationState.data.toCsv(props.dataset!);
+            download("annotations.csv", "data:text/csv;charset=utf-8," + encodeURIComponent(csvData));
+            console.log(csvData);
+          }}
+        >
+          Export CSV
+        </TextButton>
+      </FlexRow>
+
+      {/* Label selection and edit/create/delete buttons */}
+      <FlexRow $gap={6} style={{ width: "100%" }}>
+        <FlexRow $gap={6}>
+          <VisuallyHidden id={LABEL_DROPDOWN_LABEL_ID}>Current label</VisuallyHidden>
+          <SelectionDropdown
+            selected={(currentLabelIdx ?? -1).toString()}
+            items={selectLabelOptions}
+            onChange={onSelectLabelIdx}
+            disabled={currentLabelIdx === null}
+            showSelectedItemTooltip={false}
+            htmlLabelId={LABEL_DROPDOWN_LABEL_ID}
+          ></SelectionDropdown>
+
+          {/*
+           * Hide edit-related buttons until edit mode is enabled.
+           * Note that currentLabelIdx will never be null when edit mode is enabled.
+           */}
+          {isAnnotationModeEnabled && currentLabelIdx !== null && (
+            <LabelEditControls
+              onCreateNewLabel={onCreateNewLabel}
+              onDeleteLabel={onDeleteLabel}
+              setLabelColor={(color: Color) => setLabelColor(currentLabelIdx, color)}
+              setLabelName={(name: string) => setLabelName(currentLabelIdx, name)}
+              selectedLabel={selectedLabel}
+              selectedLabelIdx={currentLabelIdx}
+              selectionMode={props.annotationState.selectionMode}
+              setSelectionMode={props.annotationState.setSelectionMode}
+            />
+          )}
+        </FlexRow>
 
+        {/* View mode selection */}
+        <label>
+          <VisuallyHidden>View mode</VisuallyHidden>
           <Radio.Group
-            buttonStyle="solid"
             style={{ display: "flex", flexDirection: "row" }}
             value={viewType}
             onChange={(e) => startTransition(() => setViewType(e.target.value))}
           >
             <Tooltip trigger={["hover", "focus"]} title="Table view" placement="top">
-              <Radio.Button value={AnnotationViewType.TABLE}>
+              <Radio.Button value={AnnotationViewType.TABLE} style={{ padding: "0 8px" }}>
                 <TableOutlined />
                 <VisuallyHidden>Table view {viewType === AnnotationViewType.TABLE ? "(selected)" : ""}</VisuallyHidden>
               </Radio.Button>
             </Tooltip>
             <Tooltip trigger={["hover", "focus"]} title="List view" placement="top">
-              <Radio.Button value={AnnotationViewType.LIST}>
+              <Radio.Button value={AnnotationViewType.LIST} style={{ padding: "0 8px" }}>
```

Comment:
is there a more elegant way of doing these little style tweaks or is this pretty standard?

Reply:
I could make a styled component (using `styled-components`) which would move it up into a little CSS block. If there's a lot of styling or complex styling I usually will.